app/controllers/libraries_controller.rb

```diff
@@ -0,0 +1,45 @@
+class LibrariesController < ApplicationController
```

Comment:
Should we have namespaced this inside `Admin`?

Reply:
I thought about this looking at their file structure. I was thinking the `libraries` and other `super_admin` files might even get their own namespace? Might be a Jim question.

Reply:
I vote for a SuperAdmin namespace, and then to merge this.